system/jlib/jiface.hpp

```diff
@@ -34,19 +34,19 @@ void jlib_decl raiseAssertCore(const char *assertion, const char *file, unsigned
 
 #ifdef _PREFAST_
  #pragma warning (disable : 6244)   // generates too much noise at the moment
- #define assertex(p) ((p) ? ((void) 0) : (( (void) raiseAssertException(#p, __FILE__, __LINE__), __analysis_assume(p))))
- #define assert(p) ((p) ? ((void) 0) : (( (void) raiseAssertCore(#p, __FILE__, __LINE__), __analysis_assume(p))))
+ #define assertex(p) (likely(p) ? ((void) 0) : (( (void) raiseAssertException(#p, __FILE__, __LINE__), __analysis_assume(p))))
```

Comment:
I think __analysis_assume is a visual studio feature, primarily used to help a static analysis tool. (Used by PREfast)